### DISCUSSION TITLE
Upgrade

### DISCUSSION THREAD

**Discussion by Martl91** [pytr-org/pytr#84] [Q&A] [answered]
Hi, i am new in this world and use this option for downloading the files. I have 3 questions.
1. How i can see the upgrade is done? When is do it with pipx upgrade pytr - the result is: "version is already the latest"
when i use: pipx upgrade-all : he is doing something but i dont know what?
After i go to util.py and check the conten as examlpe line 150 is not the same like in the issue description.
2. With the older version i can download files till last week and one from today but not between. When i delete all files in the folder he will create all new,  say downloaded but i can't find few documents. 
The files with green mark are ok, the files with blue line are too new and now dokument till now also ok, but the red ones , i can't find.
![Screenshot 2024-07-02 223101](https://github.com/pytr-org/pytr/assets/174526419/6eaa8c2a-627f-49b5-a508-f05bbf83d1ee)

3. What is the correct way to get the lates util.py ?
Perhaps i do something wrong and wit would be great for little help.
Thank you Martl91

**Comment by fredo-47** [comment]
This is how updating worked for me:
https://github.com/pytr-org/pytr/issues/79#issuecomment-2203889979

**Reply by Martl91** [reply]
Ok is solved, i forgot after Git installation to restart th eComputer.. after restart it works.. i dont delet this answer because it could be other have same problems.. :-) 
With this workaround from you all documents are available. 
Hi fredo, thank you for this information but i cant fix it: i dont know why i always get this result.
![Screenshot 2024-07-03 005245](https://github.com/pytr-org/pytr/assets/174526419/c651bef4-1667-4e90-8e76-cf826be3a0ef)
 I tried also some other path but dont know what is the problem. Perhaps you can give me a manual how to use it?
Thank you

**Comment by Katzmann1983** [comment]
Just released version 0.2.0.
Please try 
´´´
pipx upgrade pytr
pytr -V 
´´

**Reply by Martl91** [reply]
HI, i tried but i get not the result which i want: i deinstalles pytr , new installation .. tried upgrade but always the same result under pytr -V  0.1.9 
![Screenshot 2024-07-03 173215](https://github.com/pytr-org/pytr/assets/174526419/c1daa577-1b0a-46d5-bbca-d3f551a44b16)
or is it correct?
Thank you

**Reply by Katzmann1983** [reply, marked as answer]
There is still an older version of pytr installed, which is used before the pip version. Try
where pytr

**Reply by Martl91** [reply]
Thank you , yes there were 2 pytr , i uninstalled both , and install it again,  then it works . i get all files now.
